tf_encrypted/convert/register.py

```diff
@@ -308,13 +323,17 @@ def _strided_slice(converter, node: Any, inputs: List[str]) -> Any:
   end = tf.constant(end.attr["value"].tensor)
   strides = tf.constant(strides.attr["value"].tensor)
 
-  return tfe.strided_slice(input_out, begin, end,
-                                          strides=strides,
-                                          begin_mask=begin_mask,
-                                          end_mask=end_mask,
-                                          ellipsis_mask=ellipsis_mask,
-                                          new_axis_mask=new_axis_mask,
-                                          shrink_axis_mask=shrink_axis_mask)
+  return tfe.strided_slice(
```

Comment:
wow this is a huge improvement, not sure how this one was so bad before

Reply:
yeah, column alignment pure sucks :D